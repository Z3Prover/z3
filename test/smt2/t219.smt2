
; Copyright (c) 2015 Microsoft Corporation


(declare-const x Int)
(declare-const z Int)

(assert (forall ((y Int)) (not (and (<= 0 y) (<= y x)))))
(assert (= z (+ x 1)))

(apply qe)