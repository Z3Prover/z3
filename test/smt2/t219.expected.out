(goals
(goal
  (not (>= x 0))
  (= z (+ x 1))
  :precision precise :depth 1)
)
